docs/scripts/demo-line.js

```diff
@@ -13334,10 +13335,14 @@ webpackJsonp([5,10],[
 	         * @return void
 	         */
 	        function updateTitle(dataPoint) {
-	            var date = new Date(dataPoint[dateLabel]),
-	                tooltipTitleText = title + ' - ' + formatDate(date);
+	            if (shouldShowDateInTitle) {
+	                var date = new Date(dataPoint[dateLabel]),
+	                    tooltipTitleText = title + ' - ' + formatDate(date);
 	
-	            tooltipTitle.text(tooltipTitleText);
+	                tooltipTitle.text(tooltipTitleText);
+	            } else {
+	                tooltipTitle.text(title);
+	            }
```

Comment:
They are the ES2015 template literals, you can check more here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

thanks a lot!